src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Minkowski.cpp

```diff
@@ -103,6 +103,22 @@ tnsr::ii<T, Dim> Minkowski<Dim>::extrinsic_curvature(const tnsr::I<T, Dim> &x,
 noexcept {
   return tnsr::ii<T, Dim>(make_with_value<T>(x, 0.));
 }
+
+template<size_t Dim>
+template<typename T>
+tnsr::Ijj<T, Dim> Minkowski<Dim>::christoffel_2nd_kind(const tnsr::I<T, Dim>& x,
+                                                       const double /*t*/) const
+    noexcept {
+  return tnsr::Ijj<T, Dim>(make_with_value<T>(x, 0.));
```

Comment:
You can directory `make_with_value<tnsr::Ijj<T, Dim>>`.

Reply:
Fixed this and the same change below.